inlong-sort/sort-connectors/sort-jdbc/src/main/java/org/apache/inlong/sort/jdbc/table/JdbcDynamicTableFactory.java

```diff
@@ -48,7 +48,7 @@
 
 /**
  * Copy from org.apache.flink:flink-connector-jdbc_2.11:1.13.5
- *
+ * <p>

```

Comment:
<img width="306" alt="image" src="https://user-images.githubusercontent.com/17596132/169640732-b4c441b5-59f1-4c0d-926d-d5c5a480fa55.png">

Please revise it.

Reply:
done